lib/foreman_maintain/report.rb

```diff
@@ -11,6 +11,12 @@ def sql_count(sql)
       feature(:foreman_database).query(sql).first['count'].to_i
     end
 
+    def table_exists(table)
+      subquery = "SELECT 1 FROM information_schema.tables WHERE table_name = '#{table}'"
+      sql = "SELECT EXISTS (#{subquery}) AS table_existence"
```

Comment:
Why not `SELECT count(*) from information_schema.tables WHERE table_name = '#{table}'`?
And then check if the count > 0?

Reply:
Eh, old habits die hard I guess. Generally I try to avoid counting _all the things_ when I only need to know if there's at least one. It's probably an overkill in a situation like this where there will be either none or exactly one.